### PR TITLE
pybind11: Remove SWIG remnants.

### PR DESCRIPTION
**Pull request by drmpeg**

### CODE REVIEW

File: gr-qtgui/lib/eye_sink_c_impl.cc [hunk 1]
```diff
@@ -188,7 +188,7 @@ void eye_sink_c_impl::set_samp_per_symbol(unsigned int sps)
 
 void eye_sink_c_impl::set_title(const std::string& title)
 {
-    // set_title no longer used but called by swig
```
Comment (mormj) [inline gr-qtgui/lib/eye_sink_c_impl.cc:191]: Can set_title be removed altogether?

File: docs/doxygen/other/build_guide.dox.in [hunk 2]
```diff
@@ -145,8 +144,8 @@ cmake is not case sensitive about these options. Similarly, "true",
 "on", or "yes" will turn this component on. All components are enabled
 by default so long as their dependencies are met.
 
-An example is -DENABLE_PYTHON=False turns off building any Python or
-Swigging components. The result will be the GNU Radio libraries and
+An example is -DENABLE_PYTHON=False turns off building any Python
+components. The result will be the GNU Radio libraries and
```
Comment (mormj) [inline docs/doxygen/other/build_guide.dox.in:148]: I would add "building any Python components or generating Python bindings" as they are two separate things.

Comment (drmpeg) [reply docs/doxygen/other/build_guide.dox.in]: Done.

File: gr-qtgui/lib/eye_sink_c_impl.cc (hunk 1 above)
Comment (drmpeg) [reply gr-qtgui/lib/eye_sink_c_impl.cc:191]: Yes, but it touches quite a few files. I'm thinking it should be a separate pull request.

**Review by mormj — APPROVED** [review]

**Comment by mormj** [comment]
Looks good!  @drmpeg - thanks for doing this scrub - it's something I've been meaning to do but now I'm glad you did this PR.

Merging.

**Pull request merged**